Week2/349_Intersection_of_2_arrays.cpp

```diff
@@ -0,0 +1,47 @@
+class Solution {
+public:
+    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
+        //if (nums1.size() > nums2.size()) return intersection(nums2, nums1);
```

Comment:
Because if their actual sizes are different it does not say anuthing about sizes of their unique arrays.
So I thought that this will give us profit only sometimes.